classes/ActionScheduler_AdminView.php

```diff
@@ -258,6 +288,7 @@ public static function row_actions( $actions, $post ) {
 	 * @return string The URL for running the action.
 	 */
 	private static function get_run_action_link( $action_id, $operation = 'process' ) {
```

Comment:
I did not remove any private property/function because they are still referenced from some of the public (but deprecated method). Are we breaking those functions? (while keeping the function accesible?)